lib/screens/login.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:flutter/material.dart';
+import 'package:admu_student_app/screens/onboarding.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class Login extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MediaQuery(
+      data: MediaQueryData(),
+      child: MaterialApp(
```

Comment:
`lib/main.dart` already uses a `MaterialApp` class to generate the app, having another one (nested) might lead to errors later on (?)